### DISCUSSION TITLE
How to automatically grant all users access to a project

### DISCUSSION THREAD

**Discussion by RonRademaker** [healthchecks/healthchecks#746] [Q&A]
Hi,

In our healthchecks setup we only have one project with checks and all users that login should automatically have access to that project. They also don't need the default (empty) project that is created when they log in. Is there any way to achieve this?

Thanks,
Ron

**Comment by cuu508** [comment]
Hi Ron,

There is no built-in functionality for automatically granting access to a project. One way to achieve this would be to fork the codebase and add custom logic in `hc.accounts.backends.CustomHeaderBackend` (assuming you use `REMOTE_USER_HEADER` setting for external authentication).

**Reply by RonRademaker** [reply]
Would there be a way to do this using API calls?
